api/bertymessenger.proto

```diff
@@ -264,18 +264,18 @@ message Account {
 }
 
 message Interaction {
-  string cid = 1 [(gogoproto.moretags) = "gorm:\"uniqueIndex\"", (gogoproto.customname) = "CID"];
-  AppMessage.Type type = 2;
+  string cid = 1 [(gogoproto.moretags) = "gorm:\"uniqueIndex\"", (gogoproto.customname) = "CID"]; // FIXME(?): primaryKey instead of uniqueIndex?
```

Comment:
If you find a way to have both a primary key and a secondary autoincrement index I would gladly make the change